### ISSUE TITLE
Fix accessibility issues.

### ISSUE THREAD

**Issue by maomeara63** [GSA/usagov-benefits-eligibility#337]
Assess and fix a few accessibility issues.

**Comment by maomeara63** [comment]
- [ ] "Find out how to apply" is throwing an alert for a possible heading. Check how it is coded to see if there's something different from the other list items.
- [ ] Showing multiple form labels on the "sort benefits by" dropdown on the life event pages.
- [ ] On the life event pages, showing many indications of "fieldset missing legend" which are associated with each instance of a checkbox or dropdown.
- [ ] On the life event pages, I wonder if "Sharing and printing" shouldn't be an H2, while maintaining the current smaller display. (It isn't really a subhead to the earlier H2.)
- [ ] Reading order in the cards is off. I think we might look at it together. I'm not sure what is going on.

**Comment by fpigeonjr** [comment]
1. **Find out how to apply** in the benefit card is a paragraph tag. Is this the right code @maomeara63?
<img width="1130" alt="Screen Shot 2022-04-01 at 10 57 23 AM" src="https://user-images.githubusercontent.com/4629398/161299485-8afc4bc3-652f-45b4-84b1-13455fba9db5.png">

**Comment by fpigeonjr** [comment]
[This issue](https://github.com/uswds/uswds/issues/4550) may be related to 5.

**Comment by fpigeonjr** [comment]
5. Benefit card is reading as expected using VoiceOver on Safari

https://user-images.githubusercontent.com/4629398/162068483-5e4a81ca-6e49-4d93-9705-65907ca359c9.mov

**Comment by fpigeonjr** [comment]
5. Benefit card  reading is off. Not in correct order and content is repeated.
Using NVDA on Chrome on windows10.

https://user-images.githubusercontent.com/4629398/162072512-39493242-61a4-4720-bd1c-0640dcd337e4.mov

**Comment by fpigeonjr** [comment]
The uswds team suggested we submit a github-issue for 5. So I did. 
[Github-issue#4630](https://github.com/uswds/uswds/issues/4630) submitted.

**Comment by r-bartlett-gsa** [comment]
Created a separate issue for item 5, see #397 
Will close this issue

**Comment by fpigeonjr** [comment]
reviewed this with @maomeara63 and the accordions are announcing open/close and reading content as expected.
- tested on windows chrome/firefox and NVDA
- tested on VO on mac/safari/chromium
